panel/models/audio.ts

```diff
@@ -8,8 +8,8 @@ export class AudioView extends WidgetView {
   private _blocked: boolean
   private _time: any
 
-  initialize(options: any): void {
-    super.initialize(options)
+  initialize(): void {
+    super.initialize()
```

Comment:
I'm not sure how this change in this file and other .ts files relates to a VTK pane?

Reply:
These are changes in bokeh model API necessary to get things to compile with bokeh master. There has been no dev release since the API changed so these fixes haven't yet been merged into master. I've got the same changes in my working tree.

Reply:
Ah; makes sense.